fili-core/src/main/java/com/yahoo/bard/webservice/web/handlers/workflow/DruidWorkflow.java

```diff
@@ -100,14 +100,12 @@ public DataRequestHandler buildWorkflow() {
         DataRequestHandler nonUiHandler = new AsyncWebServiceRequestHandler(nonUiWebService, mapper);
 
         // If query caching is enabled, the cache is checked before sending the request
-        if (BardFeatureFlag.DRUID_CACHE.isOn()) {
-            if (BardFeatureFlag.DRUID_CACHE_V2.isOn()) {
-                uiHandler = new CacheV2RequestHandler(uiHandler, dataCache, querySigningService, mapper);
-                nonUiHandler = new CacheV2RequestHandler(nonUiHandler, dataCache, querySigningService, mapper);
-            } else {
-                uiHandler = new CacheRequestHandler(uiHandler, dataCache, mapper);
-                nonUiHandler = new CacheRequestHandler(nonUiHandler, dataCache, mapper);
-            }
+        if (BardFeatureFlag.DRUID_CACHE_V2.isOn()) {
+            uiHandler = new CacheV2RequestHandler(uiHandler, dataCache, querySigningService, mapper);
+            nonUiHandler = new CacheV2RequestHandler(nonUiHandler, dataCache, querySigningService, mapper);
+        } else {
+            uiHandler = new CacheRequestHandler(uiHandler, dataCache, mapper);
+            nonUiHandler = new CacheRequestHandler(nonUiHandler, dataCache, mapper);
```

Comment:
Shouldn't the else here be NO cache request handler rather than the v1 cache request handler?